Src/main.c

```diff
@@ -108,8 +117,27 @@ int main(void)
   /* USER CODE BEGIN WHILE */
   while (1)
   {
+    //Starts the ADC
+    HAL_ADC_Start(&hadc1);
+    HAL_ADC_PollForConversion(&hadc1, HAL_MAX_DELAY);
```

Comment:
What does this do? I don't believe it is needed (but if you can tell me why iIm wrong I'm all ears. I just finished the bootcamp too ;)

Reply:
I don't think it's needed either then LOL, it was essentially a delay to make sure getting the ADC value goes smoothly (from what I understood).